students/urls.py

```diff
@@ -1,19 +1,24 @@
 from django.urls import path
 
+from students.services.emails import password_reset_request
 from students.views import StudentCreate, StudentUpdate, StudentDelete, StudentList, StudentSearchList, \
-    RegistrationStudent, UserLogin, LogoutStudent, send_email, password_reset_request
+    RegistrationStudent, UserLogin, LogoutStudent, ActivateUser, ActivateSentEmail, StudentProfile
 
 app_name = 'students'
 
 urlpatterns = [
-    path('', StudentList.as_view(), name='list'),
+    path('list/<str:type>', StudentList.as_view(), name='list'),
     path('create/', StudentCreate.as_view(), name='create'),
     path('update/<int:pk>/', StudentUpdate.as_view(), name='update'),
     path('delete/<int:pk>/', StudentDelete.as_view(), name='delete'),
     path('search/', StudentSearchList.as_view(), name='search'),
     path('registration/', RegistrationStudent.as_view(), name='registration'),
     path('login/', UserLogin.as_view(), name='login'),
     path("logout/", LogoutStudent.as_view(), name="logout"),
-    path('send_email/', send_email, name='send_email'),
-    path("password_reset/", password_reset_request, name="password_reset")
+    path("password_reset/", password_reset_request, name="password_reset"),
+    path('activate/<str:uidb64>/<str:token>', ActivateUser.as_view(),
```

Comment:
Traceback (most recent call last):
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\contrib\auth\mixins.py", line 71, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\views\generic\edit.py", line 194, in post
    return super().post(request, *args, **kwargs)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\views\generic\edit.py", line 142, in post
    return self.form_valid(form)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\views\generic\edit.py", line 126, in form_valid
    return super().form_valid(form)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\views\generic\edit.py", line 57, in form_valid
    return HttpResponseRedirect(self.get_success_url())
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\views\generic\edit.py", line 112, in get_success_url
    if self.success_url:
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\utils\functional.py", line 135, in __wrapper__
    res = func(*self.__args, **self.__kw)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\urls\base.py", line 86, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "D:\Hillel\lesson_14\homeworks\Homeworks\venv\lib\site-packages\django\urls\resolvers.py", line 694, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'list' with no arguments not found. 1 pattern(s) tried: ['students/list/(?P<type>[^/]+)$']

Please check this error